TACC/Core-Portal
# bug/WP-400: Fix impersonate url

## Overview

The /core/admin/impersonate urls needed to be above the /core/admin urls.

## Related

* [WP-400](https://jira.tacc.utexas.edu/browse/WP-400)


## Testing

1. Login in one tab with your user, and grant yourself superuser status
2. Login in an incognito tab with another test user
3. Go to https://cep.test/core/admin/impersonate/2 and confirm impersonation works
